source/main/utils/getRuntimeFolderPath.js

```diff
@@ -9,6 +9,11 @@ export default (platform, env, appName) => {
       return path.join(env.APPDATA, appName);
     }
     case 'linux': {
+      var DAEDALUS_DIR = env.DAEDALUS_DIR;
```

Comment:
cc @nikolaglumac

Reply:
without this change, daedalus logs go to an entirely different path from cardano logs, and then daedalus cant upload cardano logs when reporting issues